kit/ChildSession.hpp

```diff
@@ -29,6 +29,23 @@
 class Document;
 class ChildSession;
 
+class LOG_UICOMMANDS {
```

Comment:
This is the first SHOUTING_NAMES_WITH_UNDERSCORES I have ever seen across this code-base, or the LibreOffice core code. Please rename this to fit in with the rest of the code.

Reply:
I now renemed it to LogUiCommands.
It was as a macro like "LOG_WRN" .. but later i had to make it a class... and i forgot to rename it.